src/measure_theory/bochner_integration.lean

```diff
@@ -381,10 +383,13 @@ variables {α E μ}
 
 namespace simple_func
 
-lemma norm_eq_integral (f : α →₁ₛ[μ] E) : ∥f∥ = ((to_simple_func f).map norm).integral μ :=
+open Lp.simple_func
+
+lemma norm_eq_integral (f : α →₁ₛ[μ] E) :
+  ∥f∥ = ((Lp.simple_func.to_simple_func f).map norm).integral μ :=
```

Comment:
There is a lot of  `Lp.simple_func.*` in that file. Would it be better to open the namespace?

Reply:
I've opened `Lp.simple_func` and `Lp` and dropped a few prefixes.  Unfortunately, the lemmas are often named ambiguous things like `Lp.simple_func.dense_range` or `Lp.simple_func.normed_space`, which means that one can't drop `Lp.simple_func`, though one can drop the `Lp`.

`Lp.simple_func` is kind of a silly namespace, actually, because it can't be used for projection notation, so I'm not sure there's any point in having it -- maybe it should just all be in the `Lp` namespace.

Specificlly (if I understand correctly):  since `Lp.simple_func` is by definition a subgroup, when one says `(f : Lp.simple_func E p μ)` one actually gets `f : ↥(Lp.simple_func E p μ)` and the extra `↥` means that lemmas `Lp.simple_func.foo f` can't be used as `f.foo`.